vaadin-iron-list-flow/src/main/java/com/vaadin/flow/component/ironlist/IronList.java

```diff
@@ -150,6 +150,12 @@ public IronList() {
 
         template = new Element("template");
         getElement().appendChild(template);
+        /*
+         * Since IronList does not support pages, this forces the data
+         * communicator to operate with items limit, not with the page
+         * number and page size.
+         */
+        getDataCommunicator().setPagingEnabled(false);
```

Comment:
This disables the paging effectively.
By default.

The problem is : API still allows to enable it back.

It looks like `getDataCommunicator()` should wrap the real data communicator and disallow enable paging (e.g. via throwing).

Reply:
Good catch! Done